Document/0x05b-Basic-Security_Testing.md

```diff
@@ -41,29 +41,50 @@ Note: On Linux, you need to choose an SDK directory. `/opt`, `/srv`, and `/usr/l
+|----------------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Chance for bricking        | Soft bricked devices are always possible, but new firmware can typically still be flashed. Hard bricked devices are very rare.                                                                                                                                                   | Emulators can crash or become corrupt, but a new one can be created or a snapshot can be restored.                                                    |
+| Reset                      | Can be restored to factory settings or reflashed.                                                                                                                                                                                                            | Emulators can be deleted and recreated.                                                                                                               |
+| Snapshots                  | Not possible.                                                                                                                                                                                                                                                | Supported, great for malware analysis.                                                                                                                |
+| Speed                      | Much faster than emulators.                                                                                                                                                                                                                                  | Typically slow, but improvements are being made.                                                                                                      |
+| Cost                       | Typically start at $200 for a usable device. You may require different devices, such as one with or without a biometric sensor.                                                                                                                              | Only storage costs.                                                                                                                                   |
```

Comment:
Simulators might cost money

Reply:
Emulators also have license or other limitations. genymotion cant be used professionally. corollium has a license model. it will be less coz one license allowed multiple type of devices may be one at a time but possibility to create multiple where as physical device is one device per purchase.